database/db.js

```diff
@@ -0,0 +1,55 @@
+const DATABASE = 'kickstarter';
+const USERNAME = 'root';
```

Comment:
If you want to cut a couple lines here, you could just pass in your DB and username directly as arguments in line 5, instead of declaring them in lines 1 & 2 (doesn't seem like you're using them elsewhere in the file?):
const sequelize = new Sequelize({'kickstarter', 'root', {
  host: 'localhost', etc.
  } 
});

Reply:
I think I'm going to hold off on this one because I'm hoping to eventually factor out those values into and environmental config file.